src/CoreWCF.WebHttp/src/CoreWCF/Description/WebServiceEndpoint.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Text;
+using System.Xml;
+using CoreWCF.Channels;
+
+namespace CoreWCF.Description
+{
+    public abstract class WebServiceEndpoint : ServiceEndpoint
+    {
+        internal WebServiceEndpoint(ContractDescription contract, EndpointAddress address)
+            : base(contract, new WebHttpBinding(), address)
+        { }
+
+        //public HostNameComparisonMode HostNameComparisonMode
+        //{
+        //    get { return webHttpBinding.HostNameComparisonMode; }
+        //    set { webHttpBinding.HostNameComparisonMode = value; }
+        //}
+
+        public long MaxBufferPoolSize
+        {
+            get { return webHttpBinding.MaxBufferPoolSize; }
+            set { webHttpBinding.MaxBufferPoolSize = value; }
+        }
+
+        public int MaxBufferSize
+        {
+            get { return webHttpBinding.MaxBufferSize; }
+            set { webHttpBinding.MaxBufferSize = value; }
+        }
+
+        public long MaxReceivedMessageSize
+        {
+            get { return webHttpBinding.MaxReceivedMessageSize; }
+            set { webHttpBinding.MaxReceivedMessageSize = value; }
+        }
+
+        public TransferMode TransferMode
+        {
+            get { return webHttpBinding.TransferMode; }
+            set { webHttpBinding.TransferMode = value; }
+        }
+
+        public XmlDictionaryReaderQuotas ReaderQuotas
+        {
+            get { return webHttpBinding.ReaderQuotas; }
+            set { webHttpBinding.ReaderQuotas = value; }
+        }
+
+        public WebHttpSecurity Security
+        {
+            get { return webHttpBinding.Security; }
+        }
+
+        public Encoding WriteEncoding
+        {
+            get { return webHttpBinding.WriteEncoding; }
+            set { webHttpBinding.WriteEncoding = value; }
+        }
+
+        public WebContentTypeMapper ContentTypeMapper
+        {
+            get { return webHttpBinding.ContentTypeMapper; }
+            set { webHttpBinding.ContentTypeMapper = value; }
+        }
+
+        public bool CrossDomainScriptAccessEnabled
+        {
+            get { return webHttpBinding.CrossDomainScriptAccessEnabled; }
+            set
+            {
+                webHttpBinding.CrossDomainScriptAccessEnabled = value;
+            }
+        }
+
+        protected abstract Type WebEndpointType { get; }
+
+        WebHttpBinding webHttpBinding
+        {
```

Comment:
Accessibility modifier

Reply:
This was addressed in 063bb16c3698777ec49fc33d40cd832d952fd106.